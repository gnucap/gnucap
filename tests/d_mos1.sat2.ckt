nmos n gate, saturated, js specified (unreasonable value) default bulk cap
M1   2  2  0  4  cmosn  l= 9.u  w= 9.u  nrd= 1.  nrs= 1. as=81p ad=81p
Vds   3  0  5.
Rds   2  3  100.K
Vbs   4  0  0.
.model cmosn  nmos ( level=1  vto= 0.844345  kp= 41.5964u  gamma= 0.863074 
+ phi= 0.6  rd= 0.  rs= 0.  pb= 0.7  cgso= 218.971p 
+ cgdo= 218.971p  cgbo= 0.  rsh= 0.  cj= 384.4u  mj= 0.4884  cjsw= 527.2p 
+ mjsw= 0.3002  js= 10  tox= 41.8n  nsub= 15.3142E+15  nss= 1.E+12 
+ tpg=1  xj= 400.n  ld= 265.073n  uo= 503.521 
+ kf= 0.  af= 1.  fc= 0.5 )
*+(* vfb=-0.4241892 )
*    lambda= 0.00906241  vmax= 55.9035K
.print op v(nodes) iter(0)
.op
.print op i(v*) ps(v*)
.op
.print op id(m1) vgs(m1) vds(m1) vbs(m1) vth(m1) vdsat(m1)
.op
.print op gm(m1) gds(m1) gmb(m1) cbd(m1) cbs(m1)
.op
.print op cgsovl(m1) cgdovl(m1) cgbovl(m1) cgate(m1) region(m1)
.op
.print op cgs(m1) cgd(m1) cgb(m1) vgst(m1) von(m1)
.op
.print op cgst(m1) cgdt(m1) cgbt(m1) is(m1) ig(m1) ib(m1)
.op
.print op p(m1) pd(m1) ps(m1) ids(m1) idstray(m1) iderror(m1)
.op
.print op vdm(m1) vgm(m1) vbm(m1) vsm(m1)
.op
.print op vd(m1) vg(m1) vb(m1) vs(m1)
.op
.print op v(ddb.m1) i(ddb.m1) p(ddb.m1) cap(ddb.m1) r(ddb.m1) region(ddb.m1)
.op
.print op v(dsb.m1) i(dsb.m1) p(dsb.m1) cap(dsb.m1) r(dsb.m1) region(dsb.m1)
.op
*>.status notime
.end
