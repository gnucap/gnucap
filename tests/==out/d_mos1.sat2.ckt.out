nmos n gate, saturated, js specified (unreasonable value) default bulk cap 
cmosn: rsh - rs - rd conflict: using rsh
#           v(2)       v(3)       v(4)       iter(0)   
 27.        2.0069     5.         629.f      7.        
#           i(Vds)     i(Vbs)     ps(Vds)    ps(Vbs)   
 27.       -29.931u    62.853n    149.66u    0.        
#           id(M1)     vgs(M1)    vds(M1)    vbs(M1)    vth(M1)    vdsat(M1) 
 27.        29.931u    2.0069     2.0069     628.53f    0.84435    1.1625    
#           gm(M1)     gds(M1)    gmb(M1)    cbd(M1)    cbs(M1)   
 27.        51.384u    0.         28.627u    0.         0.        
#           cgsovl(M1) cgdovl(M1) cgbovl(M1) cgate(M1)  region(M1)
 27.        1.9707f    0.         0.         62.973f    7.        
#           cgs(M1)    cgd(M1)    cgb(M1)    vgst(M1)   von(M1)   
 27.        41.982f    0.         0.         1.1625     0.84435   
#           cgst(M1)   cgdt(M1)   cgbt(M1)   is(M1)     ig(M1)     ib(M1)    
 27.        43.953f    0.         0.        -29.868u    0.         62.853n   
#           p(M1)      pd(M1)     ps(M1)     ids(M1)    idstray(M1) iderror(M1)
 27.        60.068u    60.068u    0.         29.868u   -62.853n    ??        
#           vdm(M1)    vgm(M1)    vbm(M1)    vsm(M1)   
 27.        1.0034     1.0034    -1.0034    -1.0034    
#           vd(M1)     vg(M1)     vb(M1)     vs(M1)    
 27.        2.0069     2.0069     628.53f    0.        
#           v(M1.Ddb)  i(M1.Ddb)  p(M1.Ddb)  cap(M1.Ddb) r(M1.Ddb)  region(M1.Ddb)
 27.       -2.0069    -62.853n    126.14n    0.         31.93Meg  -1.        
#           v(M1.Dsb)  i(M1.Dsb)  p(M1.Dsb)  cap(M1.Dsb) r(M1.Dsb)  region(M1.Dsb)
 27.        628.53f    20.E-21    0.         0.         31.93Meg   1.        
Gnucap   System status
iterations: op=7, dc=0, tran=0, fourier=0, total=84
transient timesteps: accepted=0, rejected=0, total=0
nodes: user=3, subckt=0, model=0, total=3
dctran density=77.8%, ac density=77.8%
